### PR TITLE
JENKINS-68934 Small revamp to loading spinner design 

### PR DESCRIPTION
**Pull request by janfaracik**
see [JENKINS-68934](https://issues.jenkins.io/browse/JENKINS-68934)

This is a small revamp to the loading spinner to make it more inline with newer Jenkins' components, make it more responsive and to improve UX when showing/hiding.

**Before**
Screenshot 2022-06-23 at 12 40 24

**After**
image

**New show/hide animation when loading heavy pages (e.g. Configure project page)**

https://user-images.githubusercontent.com/43062514/175290303-b3dc9c4f-cb17-4b5b-add7-d5e5b6766c56.mp4

**What's changed**
* The spinner now responds to browser font size, so it'll scale correctly rather than being tiny
* The text is now title case (this was a holdover from the initial Jenkins UX works when certain parts of the ui were UPPERCASE)
* The spinner/page overlay will now fade in/out for heavy pages, rather than a jarring show/hide

### Proposed changelog entries

* Small revamp to loading spinner design

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6687"></a>

### CODE REVIEW

**Comment by timja** [comment]
Not sure if related but help icon looks skewed?

image

Looks good apart from that

**Review by timja — APPROVED** [review]

**Comment by NotMyFault** [comment]
Looks good overall, though various ATH failures: https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-794/1/

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
